index.html

```diff
@@ -1565,45 +1565,40 @@ <h3>Service properties</h3>
     </section>
 
     <section>
-      <h2>DID Subject</h2>
+      <h2>Identification</h2>
 
       <p>
-  The <a>DID subject</a> is the entity that the <a>DID document</a> is about.
-  That is, it is the entity identified by the <a>DID</a> and described by the
-  <a>DID document</a>.
+It is useful to be able to identify the <a>DID subject</a>, the <a>DID
+controller</a>, and any alternate identifiers that the <a>DID subject</a>
+might also be known as. The following section elaborates on the mechanisms for
+doing so in the <a>DID document</a> data model.
```

Comment:
```suggestion
It is necessary to learn the identifiers used within a <a>DID document</a> to
identify the <a>DID subject</a> and the <a>DID controller</a>, as well as any
alternate identifiers that might also be used to identify the <a>DID subject</a>.
The following section elaborates on the mechanisms the <a>DID document</a> data
model provides for so doing.
```

Reply:
Perhaps `identify` => `refer to` or `reference`.

Reply:
Applied @TallTed's suggestion with @dlongley's modification